libs.ts

```diff
@@ -860,7 +848,10 @@ export const libs: Lib[] = [
 			text: 'Has a PR',
 			url: 'https://github.com/discordeno/discordeno/pull/3502'
 		},
-		polls: 'No'
+		polls: {
+			text: 'Has a PR',
+			url: 'https://github.com/discordeno/discordeno/pull/3542'
+		}
```

Comment:
looks like this has since been merged

```suggestion
		polls: 'Dev Version'
```